crates/pallet-token-claims/src/lib.rs

```diff
@@ -9,7 +9,10 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use frame_support::traits::{Currency, StorageVersion};
+use frame_support::{
+    inherent::Vec,
```

Comment:
Why is `Vec` import added here?

Reply:
We need to import Vec explicitly as `GenesisConfig` is enabled in `no_std` now due to https://github.com/paritytech/substrate/pull/14108.

Thanks for mentioning, imported from `sp_std::prelude`.